recipes/pathy/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pathy" %}
+{% set version = "0.3.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f6a85eb21d4284503fceaa1a6767e736c8b02b57e19a3de4d83a8e054bcbf18a
+  patches: patches/0001-setup-requirements.patch
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - smart_open >=2.2.0,<4.0.0
+    - typer >=0.3.0,<1.0.0
+    - dataclasses >=0.6,<1.0  # [py<=36]
+
+test:
+  imports:
+    - {{ name }}
+
+about:
+  home: https://github.com/justindujardin/pathy
+  license: Apache-2.0
+  license_file: LICENSE
+  summary: A Path interface for local and cloud bucket storage
+  description: |
+    Pathy is a python package (with type annotations) for working with Bucket
+    storage providers. It provides a CLI app for basic file operations between
+    local files and remote buckets. It enables a smooth developer experience by
+    supporting local file-system backed buckets during development and testing.
+    It makes converting bucket blobs into local files a snap with optional
+    local file caching of blobs.
+  doc_url: https://github.com/justindujardin/pathy
+  dev_url: https://github.com/justindujardin/pathy
+
+extra:
+  recipe-maintainers:
+    - honnibal
+    - ines
+    - adrianeboyd
+    - justindujardin
```

Comment:
@ines @honnibal Are you happy to be listed as maintainers of this recipe?

Reply:
Yes, happy to be a maintainer of any project associated with our org 🙂

Reply:
Yes I consent.